alvr/graphics/src/lobby.rs

```diff
@@ -409,8 +409,8 @@ impl LobbyRenderer {
         &self,
         view_params: [LobbyViewParams; 2],
         hand_data: [(Option<DeviceMotion>, Option<[Pose; 26]>); 2],
-        body_skeleton: Option<Vec<Option<Pose>>>,
-        body_tracking_type: Option<BodyTrackingType>,
+        additional_motions: Option<Vec<DeviceMotion>>,
+        body_skeleton_data: (Option<Vec<Option<Pose>>>, Option<BodyTrackingType>),
```

Comment:
Keep the two parameters separate as before, it's clearer

Reply:
how should I handle clippy's too many arguments warning?

Reply:
Use `#[warn(clippy::too_many_arguments)]` on top of the fn signature